test/extended/templates/templateservicebroker_security.go

```diff
@@ -68,7 +69,19 @@ var _ = g.Describe("[templates] templateservicebroker security test", func() {
 		brokercli = client.NewClient(&http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}, adminClientConfig.Host+templateapi.ServiceBrokerRoot)
 
 		viewuser = createUser(cli, "viewuser", bootstrappolicy.ViewRoleName)
+		//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, we should see about removing
+		// this add-role-to-user, as it should not be required
+		// give test user access to the openshift project's templates ... now needed with provision's SAR for get on templates (previously commented out)
+		err = cli.AsAdmin().Run("policy").Args("add-role-to-user", "view", viewuser.Name, "-n", "openshift").Execute()
```

Comment:
same comment here... the SC isn't part of the flow, so we shouldn't be blocked on SC changes.

The "viewuser" should already have access to the openshift namespace via its groups(or so i'd expect) and we're passing the proper userinfo object to the TSB invocation via our broker client, so this should "just work" already.